### DISCUSSION TITLE
Welcome to holbertonschool-AirBnB_clone Discussions!

### DISCUSSION THREAD

**Discussion by asenges** [asenges/holbertonschool-AirBnB_clone#3] [Announcements]
We’re using Discussions as a place to connect with other members of our team. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other team members.
  * Welcome others and are open-minded. 
 
  To get started, comment below with an introduction of yourself and tell us about what you do in this project.

**Comment by maurolabella** [comment]
Hola Alejandro! Hice el pull y no veo cambios en la consola ! Si puedes comentame o nos conectamos por slack!

**Reply by asenges** [reply]
Comentame de vuelta si pudiste ver y probar las nuevas funcionalidades?

**Reply by maurolabella** [reply]
Si estuve sobre estas. Tengo algunas correcciones para proponerte que van en proxima actualizacion de mi branch. 
No modifique pero tengo dudas en el update. Me parece que tendr[iamos que usar try . Luego te comento.

**Reply by asenges** [reply]
Hola Mauro, no hay problema, pudieramos tener la secuencia try/exception en el update, en el create y en el destroy. Cada vez que intente escribir en la Persistencia. Puede ser!

**Reply by maurolabella** [reply]
Creo que si. Fijate en mi branch que le hice cambios. Ahora estoy peleando con un tema de referencia circular.

**Reply by asenges** [reply]
En breve me fijo, mientras tengo una duda con relacion a una linea en el constructor de la clase base.  Por cierto ya tengo la clase User pronta para crear instancias y Persistencia.

**Reply by asenges** [reply]
Mauro, quedaron prontas las modificaciones segun las revisiones que me pasaste. Adicional meti las clases que faltaban. Llegamos al 10. De vuelta fijate que ya no hay otras branches, solo main. Debido a que ya quedo todo comiteado, pusheado y en merge al main. 

paso siguiente hacete un backup de tu local y clonate el repo de vuelta, metemos testing manana y vamos puliendo cada quien en su branch como vamos. Vamo' Arriba Saludos!

**Reply by asenges** [reply]
Chequea la imagen adj.
https://ibb.co/wy301M1
